build/xtask/src/dist.rs

```diff
@@ -225,6 +225,10 @@ impl PackageConfig<'_> {
         self.dist_dir.join(name)
     }
 
+    fn default_file(&self, name: impl AsRef<Path>) -> PathBuf {
+        self.dist_dir.join("default").join(name)
```

Comment:
Is this required because the binaries are not present in the dist_file path? Or are the binaries in default_path different from the ones in dist_file path?

Reply:
I also have this question, what is the difference with the binaries under `dist/*.elf` vs the binaries under `dist/default/`?

Reply:
Taking jefe as an example, the dist/ path has 'jefe.elf' (a relocatable) and 'jefe.tmp' (an executable).  The dist/default/ path has 'jefe' (also an executable).

As far as I can tell, the '.tmp' file is generated by linking the '.elf' file (see 'link_dummy_task' in dist.rs).  This 'dummy' linking is performed to calculate the sizes of the various tasks.  Once the sizes have been ascertained, address space can be allocated to individual tasks (see 'allocate_all').  These address space assignments are then used to re-link the '.elf' files (see 'link_task') producing the final binary for that task (with final address space assignments) in the 'dist/default/' folder.

Therefore, we need to use the dist/default/ binaries for debugging purposes as those are the ones that have the correct address space assignments.